.github/workflows/docker.yml

```
@@ -2,7 +2,7 @@ name: ovn-docker-images
 
 on:
   push:
-    branches: [ master,release-1.0,release-1.1,release-1.2 ]
+    branches: [ master,release-* ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)